tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml

```diff
@@ -54,8 +54,8 @@ spec:
 
         NUM_COMPONENTS=$(jq '.components | length' "${PYXIS_FILE}")
 
-        mkdir /workdir/sboms
-        cd /workdir/sboms
+        mkdir "$(workspaces.data.path)/downloaded-sboms"
```

Comment:
you should use a subdir for this like https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/collect-mrrc-params/collect-mrrc-params.yaml#L37
The reason is *just in case* we have concurrent runs using the same workspace, we want to use subdirs per pipelinerun to prevent collisions